tokio-util/src/context.rs

```diff
@@ -124,7 +124,7 @@ impl<F> TokioContext<F> {
     }
 
     /// Obtain a reference to the handle inside this `TokioContext`.
-    pub fn handle(&self) -> &Handle {
+    pub const fn handle(&self) -> &Handle {
```

Comment:
Same here.

Reply:
This was a missed `const` constructor actually. We can safely add `pub const fn new`. The fix was lumped in with 79e6b1a.